halo2_backend/src/poly/ipa/strategy.rs

```diff
@@ -147,7 +143,8 @@ impl<'params, C: CurveAffine> VerificationStrategy<'params, IPACommitmentScheme<
     /// specific failing proofs, it must re-process the proofs separately.
     #[must_use]
     fn finalize(self) -> bool {
-        unreachable!()
+        // TODO: Verification is cheap, ZkAccel on verifier is not a priority.
```

Comment:
Should we file an issue for the TODO?

Reply:
Yes, I think we should.
We can discuss if it is worthy or not.